cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java

```diff
@@ -70,7 +71,18 @@ public LocalStreamService(StreamCoordinatorClient streamCoordinatorClient,
   protected void initialize() throws Exception {
     for (Map.Entry<Id.Namespace, StreamSpecification> streamSpecEntry : streamMetaStore.listStreams().entries()) {
       Id.Stream streamId = Id.Stream.from(streamSpecEntry.getKey(), streamSpecEntry.getValue().getName());
-      StreamConfig config = streamAdmin.getConfig(streamId);
+      StreamConfig config;
```

Comment:
Can you add a TODO here and relate it to the JIRA about atomic resource management (not sure if we have one JIRA yet)? Because the metadata store and the resources (file in this case) shouldn't be out of sync.